### DISCUSSION TITLE
Failed to stat file

### DISCUSSION THREAD

**Discussion by St0rm86** [jomjol/AI-on-the-edge-device#1599] [Q&A] [answered]
Hallo, 
ich habe bereits meinen Gaszähler und Wasserzähler mit jeweils einer ESP32 CAM ausgestattet und es funktioniert super mit der AI-on-the-edge.
Nun wollte ich auch noch meinen Stromzähler damit ausstatten, habe aber Probleme, die ich bei den ersten Beiden für Gas und Wasser nicht hatte.

Es werden mir keine Bilder angezeigt. Wenn ich Refernzen anlege und jeweils auf update Reference klicke, wird mir auch nicht die Referenz angezeigt. Es bleiben die Symbole wie im Screenshot.
![grafik](https://user-images.githubusercontent.com/26872195/207985596-04783d5f-b4c5-4e19-98d4-1acd78c0181d.png)

Komme ich zum Punkt Config, bekomme ich immer 2 Fehlermeldungen:

In the selectd field the value '/config/ana-cont_11.3.0_s2.tflite' in the section 'Analog' in the field 'Model' is invalid.
PLEASE CHECK BEFORE SAVING!

In the selectd field the value '/config/dig-cont_0570_s3.tflite' in the section 'Digits' in the field 'Model' is invalid.
PLEASE CHECK BEFORE SAVING!

Ich kann bei diesen Punkten in der Config nichts auswählen. das Auswahlmenü geht auf, ist aber leer:
![grafik](https://user-images.githubusercontent.com/26872195/207986293-11b49fb1-9cf4-4d6d-a12a-4908a74305e8.png)

Woran kann das liegen?

**Comment by caco3** [comment]
Schau mal, was das log sagt.

Allenfalls die SD-Karte neu formatieren udn die Daten neu darauf schreiben.

**Comment by St0rm86** [comment]
[      14] 2022-12-15T22:33:13	<INF>	[SNTP] Status OK
[      14] 2022-12-15T22:33:13	<INF>	[SNTP] Time is synced with NTP Server pool.ntp.org
[      14] 2022-12-15T23:33:13	<INF>	[SNTP] Time zone set to CET-1CEST,M3.5.0,M10.5.0/3
[      14] 2022-12-15T23:33:13	<INF>	[MAIN] =================================================
[      14] 2022-12-15T23:33:13	<INF>	[MAIN] ================== Main Started =================
[      14] 2022-12-15T23:33:13	<INF>	[MAIN] =================================================
[      14] 2022-12-15T23:33:13	<WRN>	[MAIN] Web UI version (?) does not match firmware version (304b9e0) !
[      16] 2022-12-15T23:33:15	<INF>	[MAIN] Initialization completed successfully!
[      20] 2022-12-15T23:33:19	<ERR>	[MQTT IF] Init with no Client_ID () or Last Will Topic (). Abort Init!
[      20] 2022-12-15T23:33:19	<ERR>	[MQTT IF] Init failed, skipping all MQTT publishings in this round!
[      10] 2022-12-15T22:34:56	<INF>	[SNTP] Status OK
[      10] 2022-12-15T22:34:56	<INF>	[SNTP] Time is synced with NTP Server pool.ntp.org
[      10] 2022-12-15T23:34:56	<INF>	[SNTP] Time zone set to CET-1CEST,M3.5.0,M10.5.0/3
[      10] 2022-12-15T23:34:56	<INF>	[MAIN] =================================================
[      10] 2022-12-15T23:34:56	<INF>	[MAIN] ================== Main Started =================
[      10] 2022-12-15T23:34:56	<INF>	[MAIN] =================================================
[      10] 2022-12-15T23:34:56	<WRN>	[MAIN] Web UI version (?) does not match firmware version (304b9e0) !
[      12] 2022-12-15T23:34:58	<INF>	[MAIN] Initialization completed successfully!
[      16] 2022-12-15T23:35:02	<ERR>	[MQTT IF] Init with no Client_ID () or Last Will Topic (). Abort Init!
[      16] 2022-12-15T23:35:02	<ERR>	[MQTT IF] Init failed, skipping all MQTT publishings in this round!
[     757] 2022-12-15T23:47:22	<ERR>	[OTA FILE] Failed to stat file: /sdcard/img_tmp/ref0.jpg!
[     757] 2022-12-15T23:47:22	<ERR>	[OTA FILE] Failed to stat file: /sdcard/img_tmp/ref0_org.jpg!
[     774] 2022-12-15T23:47:40	<ERR>	[OTA FILE] Failed to stat file: /sdcard/img_tmp/ref0_org.jpg!
[     781] 2022-12-15T23:47:46	<ERR>	[OTA FILE] Failed to stat file: /sdcard/img_tmp/ref0.jpg!
[     781] 2022-12-15T23:47:46	<ERR>	[OTA FILE] Failed to stat file: /sdcard/img_tmp/ref0_org.jpg!
[     782] 2022-12-15T23:47:48	<ERR>	[OTA FILE] Failed to stat file: /sdcard/img_tmp/ref1.jpg!
[     782] 2022-12-15T23:47:48	<ERR>	[OTA FILE] Failed to stat file: /sdcard/img_tmp/ref1_org.jpg!
[     785] 2022-12-15T23:47:51	<ERR>	[OTA FILE] Failed to stat file: /sdcard/img_tmp/ref1_org.jpg!
[     835] 2022-12-15T23:48:41	<ERR>	[OTA FILE] Failed to stat file: /sdcard/img_tmp/ref0_org.jpg!
[     835] 2022-12-15T23:48:41	<ERR>	[OTA FILE] Failed to stat file: /sdcard/img_tmp/ref0.jpg!
[    1281] 2022-12-15T23:56:06	<ERR>	[OTA FILE] Failed to stat file: /sdcard/img_tmp/ref0.jpg!
[    1281] 2022-12-15T23:56:06	<ERR>	[OTA FILE] Failed to stat file: /sdcard/img_tmp/ref0_org.jpg!

SD-Karte habe ich schon formatiert und alles neu rauf kopiert.
Es ist die gleiche SD-Karte drinne wie in den funktionierenden zweien, habe aber auch andere ausprobiert, 2GB, 16GB, 32GB. Das Problem besteht mit allen.

**Reply by caco3** [reply]
Hast du dir https://github.com/jomjol/AI-on-the-edge-device/discussions/1401 mal angeschaut? Da scheint es leider auf einiges anzukommen!

**Comment by St0rm86** [comment]
Die Karten habe ich mit Windows formatiert und auch die Dateien mit Windows kopiert.
Die ersten 2 Male hat es ja auch funktioniert und die ESP32 laufen ohne Probleme.
Beim Dritten habe ich nichts anders gemacht, das ist ja das was ich komisch finde.

Aufgefallen ist mir zudem noch:
Die Referenzen werden nicht richtig gespeichert. Wähle ich z.B. für Referenz1 ein Quadrat aus, speicher es und wähle Referenz2 aus, speicher auch dieses und gehe zurück zur Referenz1, ist dieses kein Quadrat mehr sondern ein Rechteck.

**Comment by TopGoalVl** [comment]
Könnte hier auch Issue 1597 zutreffen?

https://github.com/jomjol/AI-on-the-edge-device/issues/1597

**Reply by caco3** [reply]
> Könnte hier auch Issue 1597 zutreffen?
> 
> #1597

Nein, das ist komplett unabhängig. Dort geht es nur um die `gethost.js`

**Comment by St0rm86** [comment, marked as answer]
Das Problem trat mit der Version 12.0.1 auf, gestern habe ich es mit der neuesten Version die es hier zum Download gibt ausprobiert und es hat auf anhieb funktioniert.
